pkgs/development/libraries/prometheus-client-c/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, libmicrohttpd
+}:
+let
+  build =
+    { pname
+    , subdir
+    , buildInputs ? [ ]
+    , description
+    }:
+    stdenv.mkDerivation rec {
+      inherit pname;
+      version = "0.1.1";
+
+      src = fetchFromGitHub {
+        owner = "digitalocean";
+        repo = "prometheus-client-c";
+        rev = "v${version}";
+        sha256 = "0g69s24xwrv5974acshrhnp6i8rpby8c6bhz15m3d8kpgjw3cm8f";
+      };
+
+      nativeBuildInputs = [ cmake ];
+      inherit buildInputs;
+      doCheck = false;
```

Comment:
I don't think this is necessary. I imagine this is because it doesn't currently have tests -- if this changes in the future, we will want to run their tests without allowing the possibility of forgetting to remove this line at that point.

I removed this locally and both libraries built fine, so I'd prefer this removed (unless a strong case can be made otherwise).

```suggestion
```

Reply:
Makes perfect sense: I will commit this suggestion, and then it can be squashed in at merge time.

Reply:
Squashed and pushed locally instead.

Reply:
Thanks!

FWIW, I much prefer local squashing in a monorepo like this -- some committers make merge commits instead of squashing, which leads to those fixup commits ending up in the history which (IMO) looks ugly.

Reply:
Yep, avoidable merge commits are gross.